--- a/gcalcli/gcal.py
+++ b/gcalcli/gcal.py
@@ -1056,7 +1056,7 @@
                     all_day = self.options.get('allday')
                     try:
                         new_start, new_end = utils.get_times_from_duration(
-                            val, duration=length, all_day=all_day)
+                            val, duration=length, allday=all_day)
                     except ValueError as exc:
                         self.printer.err_msg(str(exc))
                         sys.exit(1)
@@ -1073,7 +1073,7 @@
                     new_start, new_end = utils.get_times_from_duration(
                         event['start']['dateTime'],
                         duration=val,
-                        all_day=all_day)
+                        allday=all_day)
                     event = self._SetEventStartEnd(new_start, new_end, event)
                 except ValueError as exc:
                     self.printer.err_msg(str(exc))
